locale/en/docs/guides/anatomy-of-an-http-transaction.md

```diff
@@ -126,7 +126,7 @@ request.on('error', function(err) {
 });
 ```
 
-There are other ways of [handling these errors][] such as using [`domains`][] or
+There are other ways of [handling these errors][] such as
```

Comment:
I just mean that if you have additional changes, mention the changes that already happened in the PR for the additional changes. Although now that I'm thinking about this, it's easy enough to just send a revert commit and then submit a proper PR. So, you know, if you want to set a good example and be a good citizen, might as well do that.